node-js-basics-101/node_modules/asynquence/tests.js

```diff
@@ -0,0 +1,1528 @@
+					args.unshift(testDone,label);
+					FAIL.apply(FAIL,args);
+				}
+			})
+			.then(function(){
+				clearTimeout(timeout);
+				PASS(testDone,label);
+			})
+			.onerror(function(){
+				clearTimeout(timeout);
+				var args = ARRAY_SLICE.call(arguments);
+				args.unshift(testDone,label);
+				FAIL.apply(FAIL,args);
+			});
+
+			timeout = setTimeout(function(){
+				FAIL(testDone,label + " (from timeout)");
+			},2000);
+		});
+		tests.push(function(testDone){
+			var label = "Core Test  #2", timeout;
+
+			ASQ()
+			.then(asyncDelaySeq(50))
+			.then(function(done){
+				asyncDelayFn(100)(function(){
+					done("Hello World");
+				});
+			})
+			.then(function(_,msg1){
+				clearTimeout(timeout);
+				if (msg1 === "Hello World") {
+					PASS(testDone,label);
+				}
+				else {
+					var args = ARRAY_SLICE.call(arguments);
+					args.unshift(testDone,label);
+					FAIL.apply(FAIL,args);
+				}
+			})
+			.or(function(){
+				clearTimeout(timeout);
+				var args = ARRAY_SLICE.call(arguments);
+				args.unshift(testDone,label);
+				FAIL.apply(FAIL,args);
+			});
+
+			timeout = setTimeout(function(){
+				FAIL(testDone,label + " (from timeout)");
+			},2000);
+		});
+		tests.push(function(testDone){
+			var label = "Core Test  #3", timeout;
+
+			ASQ()
+			.then(function(done){
+				asyncDelayFn(100)(function(){
+					done("Hello");
+				});
+			})
+			.then(function(done,msg1){
+				asyncDelayFn(100)(function(){
+					done(msg1,"World");
+				});
+			})
+			.then(function(_,msg1,msg2){
+				clearTimeout(timeout);
+				if (msg1 === "Hello" && msg2 === "World") {
+					PASS(testDone,label);
+				}
+				else {
+					var args = ARRAY_SLICE.call(arguments);
+					args.unshift(testDone,label);
+					FAIL.apply(FAIL,args);
+				}
+			})
+			.or(function(){
+				clearTimeout(timeout);
+				var args = ARRAY_SLICE.call(arguments);
+				args.unshift(testDone,label);
+				FAIL.apply(FAIL,args);
+			});
+
+			timeout = setTimeout(function(){
+				FAIL(testDone,label + " (from timeout)");
+			},2000);
+		});
+		tests.push(function(testDone){
+			var label = "Core Test  #4", timeout;
+
+			ASQ()
+			.then(function(done){
+				asyncDelayFn(100)(function(){
+					done.fail("Hello","World");
+				});
+			})
+			.then(function(){
+				clearTimeout(timeout);
+				var args = ARRAY_SLICE.call(arguments);
+				args.unshift(testDone,label);
+				FAIL.apply(FAIL,args);
+			})
+			.or(function(msg1,msg2){
+				clearTimeout(timeout);
+				if (msg1 === "Hello" && msg2 === "World") {
+					PASS(testDone,label);
+				}
+				else {
+					var args = ARRAY_SLICE.call(arguments);
+					args.unshift(testDone,label);
+					FAIL.apply(FAIL,args);
+				}
+			});
+
+			timeout = setTimeout(function(){
+				FAIL(testDone,label + " (from timeout)");
+			},2000);
+		});
+		tests.push(function(testDone){
+			var label = "Core Test  #5", timeout;
+
+			ASQ()
+			.then(function(done){
+				done.a.b; // throwing JS error to make sure it's caught and propagated
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.

<!-- Comment by [SideCI](https://sideci.com) -->